Content.Shared/GameObjects/Components/ActionBlocking/SharedCuffedComponent.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Content.Shared.GameObjects.EntitySystems;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization;
+using Robust.Shared.Maths;
+using Robust.Shared.ViewVariables;
+using System;
+
+namespace Content.Shared.GameObjects.Components.ActionBlocking
+{
+    public class SharedCuffedComponent : Component, IActionBlocker
+    {
+        public override string Name => "Cuffed";
+        public override uint? NetID => ContentNetIDs.CUFFED;
+
+        [ViewVariables]
+        public bool CanStillInteract = true;
```

Comment:
Wouldn't it would have made more sense making this `IsCuffed` or something?

Reply:
in the context of people with more than 2 arms it got confusing, since a player with 4 arms and 2 arms cuffed is technically cuffed but still able to interact